sys/net/gnrc/application_layer/uhcpc/gnrc_uhcpc.c

```diff
@@ -86,6 +77,9 @@ void uhcp_handle_prefix(uint8_t *prefix, uint8_t prefix_len, uint16_t lifetime,
     idx = gnrc_netif_ipv6_add_prefix(wireless, (ipv6_addr_t *)prefix, prefix_len,
                                      lifetime, lifetime);
     if (idx >= 0) {
+        /* start advertising subnet obtained via UHCP */
+        gnrc_ipv6_nib_change_rtr_adv_iface(wireless, true);
+        /* configure this router as RPL root */
```

Comment:
Would you say the same check could be applied here? Or is it safe to assume that `uhcp_handle_prefix` will only be used for "actual" prefixes?

Otherwise: LGTM :)

Reply:
UHCP is only used to push prefixes and `uhcp_handle_push` already checks for a sufficient prefix length.

Reply:
Okay, great :) Then I guess you could squash?